pysatSeasons/tests/test_core.py

```diff
@@ -0,0 +1,78 @@
+"""Test the pysat _core code."""
+
+import xarray as xr
+
+import pysat
+from pysat import instruments as pinsts
+import pysatSeasons as ps
```

Comment:
```suggestion
import pysatSeasons as pyseas
```

A more unique and consistent nickname?